### DISCUSSION TITLE
Migrate a single P SKU -> F SKU cross region

### DISCUSSION THREAD

**Discussion by Fredriksolhem** [microsoft/semantic-link-labs#196] [Q&A]
Hi!
We are about to move from a P SKU to a F SKU but we have to change region, is there any plans for Semantic links to help moving from P to F SKU in different regions?

**Comment by m-kovalsky** [comment]
Cross region migration is more complicated and at the moment not all the pieces are there to make it happen. It is on our radar as this has come up quite often.
